docs/_articles/en/einstein/einstein-setup.md

```diff
@@ -44,6 +45,10 @@ The Einstein for Developers status icon in the bottom panel of the Visual Studio
 
 You can disable or uninstall the Einstein for Developers extension at any time. Bring up the Extensions view by clicking on the Extensions icon in the Activity Bar on the side of VS Code or the **View: Show Extensions** command (⇧⌘X). Search for "Einstein for Developers" in installed extensions list, and then select **Disable** or **Uninstall**.
 
+### Use Einstein for Developers in Code Builder
+
+Einstein for Developers is disabled by default in new Code Builder environments. To use this extension in a new environment, go to **Settings** > **Application** > **Telemetry** and set the dropdown value to `all`, and then enable the extension. 
```

Comment:
It might be helpful to document how to enable the extension too. In W-15735110, there's draft text that says:
enable it through Settings → Extensions → Einstein for Developers

I don't see that setting in my existing instance of Code Builder so maybe it's new?

Reply:
It's doc-ed?